ngraph/core/include/ngraph/node.hpp

```diff
@@ -483,6 +483,9 @@ namespace ngraph
             return m_op_annotations;
         }
 
+        void set_opset_version(std::string opset_version) { m_opset_version = opset_version; }
```

Comment:
Operation can be included in several opsets. Why do we need these methods?

The original issue was about the operation version (not opset version).
In C++ we have the operation version and operation type.

Reply:
I see, that is a misunderstanding on my side. Is it better now?